app/src/server/api/external/v1Api.router.ts

```diff
@@ -334,6 +301,149 @@ export const v1ApiRouter = createOpenApiRouter({
 
       return { status: "ok" };
     }),
+  updateLogTags: openApiProtectedProc
+    .meta({
+      openapi: {
+        method: "POST",
+        path: "/update-log-tags",
+        description: "Update tags for logged calls matching the provided filters",
+        protect: true,
+      },
+    })
+    .input(
+      z.object({
+        filters: z
+          .union([
+            z.object({
+              field: z.union([z.enum(["model", "completionId"]), z.string()]),
+              equals: z.union([z.string(), z.number(), z.boolean()]),
+            }),
+            z.object({
```

Comment:
I think we had discussed only supporting `equals` for the moment and waiting for someone to ask for this? With our current db setup it would be trivial to DoS us if you start relying on `contains` and calling it in a loop.

Reply:
Okay, I can remove contains.